web/src/api/schema.d.ts

```diff
@@ -157,8 +157,6 @@ export interface components {
   pathItems: never;
 }
 
-export type $defs = Record<string, never>;
-
```

Comment:
Uhm, what is going on here?

Reply:
I ran `cargo xtask gen-openapi` according to the README as the commit hooks yelled at me

Reply:
Ok haha